CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ SET( ${PROJECT_NAME}_VERSION_PATCH 0 )
 FIND_PACKAGE( ILCUTIL REQUIRED COMPONENTS ILCSOFT_CMAKE_MODULES )
 
 # load default settings from ILCSOFT_CMAKE_MODULES
-INCLUDE( ilcsoft_default_settings )
+#INCLUDE( ilcsoft_default_settings )
```

Comment:
Quick question for whoever has time. I always have to comment this out when compiling this locally with other packages due to an otherwise multiply defined `unistall` make target, although I seem to see a protection against this in the main make files included, but apparently it's not working well. Anyone else having these problems?
I can of course revert this change for the PR, since this should not be changed in the main `mc` branch.

Reply:
@pandreetto is looking into this

Reply:
What do you mean when you write "compiling with other packages"?
Are you trying to merge the compilation of multiple components somehow?
If I remove the instruction above, cmake reports an error (Unknown CMake command "ADD_SHARED_LIBRARY")

Reply:
Hi @pandreetto , I've posted here the instructions to reproduce the problem:
https://github.com/MuonColliderSoft/LCTuple/issues/4
(it happens for all packages)